src/Components/Search/SearchBar.tsx

```diff
@@ -107,6 +107,8 @@ const SuggestionContainer = ({ children, containerProps, preview }) => {
 @track()
 export class SearchBar extends Component<Props, State> {
   public input: HTMLInputElement
+  private containerRef: any
```

Comment:
I _think_ this can be improved from `any` and typed as `React.RefObject<HTMLDivElement>` (or something like that).

Reply:
Yes, thank you! Fixing.

Reply:
It ended up being just a `Node`. Not `React.RefObject<Node>`, just a plain old `Node`. 🤷‍♀️